vshn/website
# feat: update the Events page

**Describe what changes this pull request brings**
 This pull request brings the update of the Events page


**Describe any additional changes**


**Steps to test**
1. Open [the page](https://events-update--vshn.netlify.app/events/)
2. Confirm that everything looks and works as expected


**Screenshots**
![image](https://user-images.githubusercontent.com/48465000/124594203-33aef080-de68-11eb-91e9-329b16518c4d.png)
![image](https://user-images.githubusercontent.com/48465000/124594252-41fd0c80-de68-11eb-8d2e-ee4d19e236dc.png)



**Checklist**



**TODO**
- check changes on pixel perfect


**References**
[Event Date Sorting](https://www.notion.so/pixelpoint/Event-Date-Sorting-8cbbcf504db1496aaeb703ed3d73cf3e)
